Loop/src/loop.hxx

```diff
@@ -527,6 +531,105 @@ public:
     } // for rank
   }
 
+  // Loop over the outermost "boundary" points in the interior. They correspond
+  // to points that are shifted inwards by = cctk_nghostzones[3] from those that
+  // CarpetX identifies as boundary points. From the perspective of CarpetX (or
+  // AMReX), these do not belong in the outer boundary, but rather the interior.
+  // This excludes ghost faces, but includes ghost edges/corners on non-ghost
+  // faces. Loop over faces first, then edges, then corners. Modified from
+  // loop_bnd.
+  template <int CI, int CJ, int CK, int VS = 1, int N = 1, typename F>
+  inline CCTK_ATTRIBUTE_ALWAYS_INLINE void
+  loop_int_bnd(const vect<int, dim> &group_nghostzones, const F &f) const {
```

Comment:
I have been thinking about the name a bit, and I now think that `int_bnd` would be confusing. This sounds as if this loop was looping over the interior and the boundary points, which it isn't. Can you change this to `outermost_int` or similar?

Reply:
Hmm, I see where it could be confusing. I have renamed the loops to `outermost_int`.